yarn.lock

```diff
@@ -5829,6 +5829,25 @@ babel-polyfill@6.26.0:
     core-js "^2.5.0"
     regenerator-runtime "^0.10.5"
 
+babel-preset-cozy-app@2.0.1, babel-preset-cozy-app@^2.0.1:
```

Comment:
What is the change thant brings `babel-preset-cozy-app@2.0.1` in? 🤔

Reply:
yea I can't really say to be honest, I'm going to investigate